charts/crljanitor/values.yaml

```diff
@@ -37,6 +37,9 @@ vault:
   # vault.adminUserFilePath -- (string) Vault path prefix for admin user list. Required if vault.enabled.
   # Use the same users as admin for all env by default. Override if needed in helmfile repo.
   adminUserFilePath: config/terra/crl-janitor/common/iam
+  # vault.azureManagedAppPublisherPath -- (string) Vault path for Azure managed app publisher credentials.
+  # Uses the same path for all envs by default. Override if needed in helmfile repo.
+  azureManagedAppPublisherPath: secret/dsde/terra/azure/common/managed-app-publisher
```

Comment:
Is this a good temp vault path? Open to other suggestions :)

Reply:
works for me